libavfilter/vf_libvmaf.c

```diff
@@ -46,6 +46,10 @@
 #include "libavutil/hwcontext_cuda_internal.h"
 #endif
 
+typedef struct MetaStruct {
+    AVDictionary **metadata;
+} MetaStruct;
+
```

Comment:
I believe we need this type of struct but I am not sure about what we should put inside in addition to `metadata`.

Reply:
Removed that scructure, Instead of I am using only one AVDictionary.